plutus-playground-server/usecases/Vesting.hs

```diff
@@ -178,15 +179,18 @@ contractAddress vst = Ledger.scriptAddress (validatorScript vst)
 
 -}
 
-vestFunds :: (Monad m, WalletAPI m) => Vesting -> m ()
-vestFunds vst = do
-    let amt = totalAmount vst
+vestFunds :: (Monad m, WalletAPI m) => VestingTranche -> VestingTranche -> Wallet -> m ()
+vestFunds tranche1 tranche2 ownerWallet = do
+    let vst = Vesting tranche1 tranche2 (walletPubKey ownerWallet)
```

Comment:
A type parameter for the owner would be nice but I'm not sure whether that conforms with the policy of keeping the example code as simple as possible